go.mod

```diff
@@ -73,6 +73,8 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require github.com/pkg/errors v0.9.1
```

Comment:
Why did you use an archived package?

Please do not add an additional line here, but add it to the block above.

Reply:
I ran `make tidy` as `pull-gardener-unit` was failing and the logs instructed me to do so. Additionally, the package was imported by accident. I will address this

Reply:
Import "errors" instead of "github.com/pkg/errors".

Reply:
Done, let's see if the check passed now.